src/ts/features/block-quotes.ts

```diff
@@ -0,0 +1,51 @@
+import {Feature, Settings} from '../utils/settings'
+
+export const config: Feature = {
+    id: 'blockquotes',
+    name: 'Blockquote Support',
+    settings: [{type: 'string', id: 'blockquote_prefix', initValue: '> ', label: 'Blockquote Prefix'}],
+    description:
+        'If you start a block with these characters, custom styles will be added to show the block as a blockquote.',
```

Comment:
I don't believe this description is actually being displayed anywhere - it would be good to get it shown in the menu or at least (or in addition to) when you select the item and are viewing the settings for it.  I tried to make that happen but I wasn't sure of it and didn't want to tackle it as part of this PR.